hashicorp/dev-portal
# style: try hcp callouts dark

## 🔗 Relevant links



- [Preview link](https://dev-portal-git-kstry-hcp-callouts-dark-hashicorp.vercel.app/waypoint) 🔎
- [Asana task](https://app.asana.com/0/1203652232454021/1204108582255302) 🎟️
- [Figma file](https://www.figma.com/file/v0Hyqvjdf6QJAvRkdIBV24/Dark-Mode?node-id=513-32631&t=xA88yKRiEIdhirlz-0)

## 🗒️ What

This PR updates the background callout graphic svg to have a transparent background so that the card color can serve as the base. Additionally, it pulls out the linear gradient from the svg for the compact callout and reimplements it with css. 

<img width="958" alt="Screenshot 2023-03-17 at 11 07 42 AM" src="https://user-images.githubusercontent.com/36613477/225984465-53d8b496-3ab8-4883-9a2a-7b13dd1d10e8.png">

<img width="382" alt="Screenshot 2023-03-17 at 10 49 20 AM" src="https://user-images.githubusercontent.com/36613477/225980643-dd4d634c-e2f2-4406-b5ae-5ac791729cc8.png">


## 📸 Design Screenshots

<img width="771" alt="Screenshot 2023-03-17 at 10 48 59 AM" src="https://user-images.githubusercontent.com/36613477/225980548-9920ae71-a4c5-44b8-8545-f93b0fc54bfe.png">


## 🧪 Testing

- Check a [product landing page](https://dev-portal-git-kstry-hcp-callouts-dark-hashicorp.vercel.app/waypoint)
- Check an[ install page](https://dev-portal-git-kstry-hcp-callouts-dark-hashicorp.vercel.app/waypoint/downloads)

## 💭 Anything else?
